src/namespaces/PlaybackNamespace.js

```diff
@@ -189,9 +195,8 @@ export default class PlaybackNamespace extends GMusicNamespace {
             // Make sure that this is the first of the notifications for the
             // insertion of the song information elements.
             if (!currentTrack.equals(lastTrack)) {
-              this.emit('change:track', currentTrack);
```

Comment:
😢 Idm about massive amounts of commits.  I use the "squash and merge" button on github anyway so it will end up being a single commit 👍